stack.yaml

```diff
@@ -131,3 +142,11 @@ ghc-options:
   cardano-sl:            -Werror
   cardano-sl-tools:      -Werror
   cardano-sl-lwallet:    -Werror
+  cardano-sl-explorer:   -Werror
+
+# Added for explorer
+flags:
+  cardano-sl:
+    with-explorer: True
```

Comment:
Under `stack.yaml` packages I added `explorer` - which pulls the `cardano-sl-explorer.cabal` and everything in it. That means it tries to build `explorer` by default, and if we disable the flag `with-explorer` then it still builds but it raises the gold-linker issue we are familiar with.